components/VoteCard.jsx

```diff
@@ -13,29 +26,31 @@ export default function VoteCard({ question, inputs }) {
 
       <Header pageName={"Vote"} />
       <StyledP>{question}</StyledP>
-
-      {inputs.map((voteListItem) => {
-        return (
-          <ul key={voteListItem.id}>
-            <ListItem>
-              <button>👎</button>
-              {voteListItem.value}
-              <button>👍</button>
+      <ul>
+        {inputs.map((voteListItem) => {
+          return (
+            <ListItem key={voteListItem.id}>
+              <button onClick={() => handleVoteMinus(voteListItem.id)}>
+                <img src="/Icon/.svg" height="25px" width="25px" />
+              </button>
+              <p>{voteListItem.value}</p>
+              <button onClick={() => handleVotePlus(voteListItem.id)}>
```

Comment:
You could use destructuring here for more consice code:

```suggestion
        {inputs.map(({ id, value }) => {
          return (
            <ListItem key={id}>
              <button onClick={() => handleVoteMinus(id)}>
                <img src="/Icon/.svg" height="25px" width="25px" />
              </button>
              <p>{value}</p>
              <button onClick={() => handleVotePlus(id)}>
```